### PR TITLE
DOCS-2124 Creating tenants via UI

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/installation/multi-tenant/configure-accounts.md [hunk 1]
```diff
@@ -73,9 +73,30 @@ After getting your request, the Wallarm technical support will:
...
+* **Partner** - to represent an independent company that installs a filtering node within its system infrastructure to provide it's clients with attack mitigation.
+* **Multi-tenant customer** - to represent an independent company that wants to isolate the data on protected environments from each other by creating further sub-tenants.
+* **Ordinary tenant** (default) - to represent:
+
+    * An independent company that does not plan to isolate the data on protected environments from each other and thus does not plan to create further sub-tenants.
+    * An isolated environment itself.
+
+You can create a new **Administrator** user for your new tenant via the **Create a new user** option. The invitation email will be sent to the specified address.
+
+For the newly created tenant
```
Comment (AnastasiaTWW) [inline docs/latest/installation/multi-tenant/configure-accounts.md:95]: ?

File: docs/latest/installation/multi-tenant/configure-accounts.md [hunk 2]
```diff
@@ -73,9 +73,30 @@ After getting your request, the Wallarm technical support will:
...
-To create the tenant, it is required to send authenticated requests to Wallarm API. Authenticated requests to Wallarm API can be sent from your own API client or from the [Wallarm API Console](../../api/overview.md) that defines the authentication method:
+#### Via Wallarm Console
+
+Under the **Global administrator** account, you can create tenants via Wallarm Console → tenant selector → **Create tenant**.
+
+![!Creating tenant via Wallarm Console](../../images/partner-waf-node/tenant-create-via-ui.png)
+
+Tenants can represent [different entities](overview.md), thus select:
+
+* **Partner** - to represent an independent company that installs a filtering node within its system infrastructure to provide it's clients with attack mitigation.
```
Comment (AnastasiaTWW) [inline docs/latest/installation/multi-tenant/configure-accounts.md:86]: if the UI allows to create a partner or a multi-tenant customer, is the step with asking the support team to create it extra?

Comment (MaximBashurov) [reply docs/latest/installation/multi-tenant/configure-accounts.md:86]: > step with asking the support team to create it

We do not seem to have such a step.

File: docs/latest/installation/multi-tenant/configure-accounts.md [hunk 3]
```diff
@@ -73,9 +73,19 @@ After getting your request, the Wallarm technical support will:
...
+![!Creating tenant via Wallarm Console](../../images/partner-waf-node/tenant-create-via-ui.png)
+
+You can create a new **Administrator** user for your new tenant. The invitation email will be sent to the specified address.
+
+#### Via the Wallarm API
+
+To create the tenant, you can send authenticated requests to Wallarm API. Authenticated requests to Wallarm API can be sent from your own API client or from the [Wallarm API Console](../../api/overview.md) that defines the authentication method:
 
 * For requests to be sent from the **Wallarm API Console**, it is required to sign in to Wallarm Console with the **Global administrator** user role and update the Wallarm API Console page available at:
     * https://apiconsole.us1.wallarm.com/ for the US Cloud
```
Comment (AnastasiaTWW) [inline docs/latest/installation/multi-tenant/configure-accounts.md:91]: the schreenshot shows the name "Testing environment". what is this name? tenant selector shows different names of accounts

Comment (AnastasiaTWW) [reply docs/latest/installation/multi-tenant/configure-accounts.md:91]: texts On the screesnhot are bad. proofread them

Comment (AnastasiaTWW) [reply docs/latest/installation/multi-tenant/configure-accounts.md:91]: it is absolutely unclear what to specify a user for

File: docs/latest/installation/multi-tenant/configure-accounts.md (hunk 2 above)
Comment (AnastasiaTWW) [reply docs/latest/installation/multi-tenant/configure-accounts.md]: then why is it still here?

File: docs/latest/installation/multi-tenant/configure-accounts.md (hunk 3 above)
Comment (MaximBashurov) [reply docs/latest/installation/multi-tenant/configure-accounts.md:91]: > the schreenshot shows the name "Testing environment". what is this name? tenant selector shows different names of accounts

"Testing environment" is just typed in name. It is not selected from anywhere. The remaining part of this comment is outdated as the screen has changed.

Comment (MaximBashurov) [reply docs/latest/installation/multi-tenant/configure-accounts.md:91]: > texts On the screesnhot are bad. proofread them

Screen has changed, is this still actual? IMHO, current texts are ok.

Comment (MaximBashurov) [reply docs/latest/installation/multi-tenant/configure-accounts.md:91]: > it is absolutely unclear what to specify a user for

Current explanation: "You can create a new Administrator user for your new tenant. The invitation email will be sent to the specified address." - is this not enough?

File: docs/latest/installation/multi-tenant/configure-accounts.md [hunk 4]
```diff
@@ -73,9 +73,19 @@ After getting your request, the Wallarm technical support will:
...
+### Step 3: Create the tenant
 
-To create the tenant, it is required to send authenticated requests to Wallarm API. Authenticated requests to Wallarm API can be sent from your own API client or from the [Wallarm API Console](../../api/overview.md) that defines the authentication method:
+#### Via Wallarm Console
+
+Under the **Global administrator** account, you can create tenants via Wallarm Console → tenant selector → **Create tenant**.
+
+![!Creating tenant via Wallarm Console](../../images/partner-waf-node/tenant-create-via-ui.png)
+
+You can create a new **Administrator** user for your new tenant. The invitation email will be sent to the specified address.
```
Comment (AnastasiaTWW) [inline docs/latest/installation/multi-tenant/configure-accounts.md:84]: add the link to the docs on users

File: docs/latest/installation/multi-tenant/configure-accounts.md (hunk 3 above)
Comment (AnastasiaTWW) [reply docs/latest/installation/multi-tenant/configure-accounts.md:91]: on the screenshot, the text explaining optional data for a user names the role as "Admin". but it is "Administrator", let's be consistent

**Pull request merged**